tap_linkedin_ads/schemas/campaigns.json

```diff
@@ -247,7 +247,7 @@
 			"additionalProperties": false,
 			"properties": {
 				"amount": {
-					"type": ["null", "string"]
+					"type": ["null", "number"]
```

Comment:
We would want to add `"multipleOf"` data to these as well to ensure they are read throughout our systems as BigDecimal types. Ex: https://github.com/singer-io/tap-shopify/blob/1162e82666ade95ce4c0ea1c6637e6500aa8ea87/tap_shopify/schemas/transactions.json#L50

Reply:
OK - I added `multipleOf` to the schema json number fields